commands/link/__tests__/link-command.test.js

```diff
@@ -192,44 +192,55 @@ Array [
       await lernaLink(testDir)("--contents", "build");
 
       expect(symlinkedDirectories(testDir)).toMatchInlineSnapshot(`
-Array [
-  Object {
-    "_src": "packages/package-1/build",
-    "dest": "packages/package-2/node_modules/@test/package-1",
-    "type": "junction",
-  },
-  Object {
-    "_src": "packages/package-1/build",
-    "dest": "packages/package-3/node_modules/@test/package-1",
-    "type": "junction",
-  },
-  Object {
-    "_src": "packages/package-2/build",
-    "dest": "packages/package-3/node_modules/@test/package-2",
-    "type": "junction",
-  },
-  Object {
-    "_src": "packages/package-2/build/cli.js",
-    "dest": "packages/package-3/node_modules/.bin/package-2",
-    "type": "exec",
-  },
-  Object {
-    "_src": "packages/package-3/build",
-    "dest": "packages/package-4/node_modules/package-3",
-    "type": "junction",
-  },
-  Object {
-    "_src": "packages/package-3/build/cli1.js",
-    "dest": "packages/package-4/node_modules/.bin/package3cli1",
-    "type": "exec",
-  },
-  Object {
-    "_src": "packages/package-3/build/cli2.js",
-    "dest": "packages/package-4/node_modules/.bin/package3cli2",
-    "type": "exec",
-  },
-]
-`);
+        Array [
+          Object {
+            "_src": "packages/package-1/build",
+            "dest": "packages/package-2/node_modules/@test/package-1",
+            "type": "junction",
+          },
+          Object {
+            "_src": "packages/package-1/build",
+            "dest": "packages/package-3/node_modules/@test/package-1",
+            "type": "junction",
+          },
+          Object {
+            "_src": "packages/package-2/build",
+            "dest": "packages/package-3/node_modules/@test/package-2",
+            "type": "junction",
+          },
+          Object {
+            "_src": "packages/package-2/build/cli.js",
+            "dest": "packages/package-3/node_modules/.bin/package-2",
+            "type": "exec",
+          },
+          Object {
+            "_src": "packages/package-3/build",
+            "dest": "packages/package-4/node_modules/package-3",
+            "type": "junction",
+          },
+          Object {
+            "_src": "packages/package-3/build/cli1.js",
+            "dest": "packages/package-4/node_modules/.bin/package3cli1",
+            "type": "exec",
+          },
+          Object {
+            "_src": "packages/package-3/build/cli2.js",
+            "dest": "packages/package-4/node_modules/.bin/package3cli2",
+            "type": "exec",
+          },
+        ]
+        `);
+    });
+  });
+
+  describe("with pnpm", () => {
+    it("should throw validation error", async () => {
+      const testDir = await initFixture("pnpm");
+      const command = lernaLink(testDir)();
+
+      await expect(command).rejects.toThrow(
+        "Link is not supported with pnpm workspaces, since pnpm will automatically link dependencies during `pnpm install`. See the pnpm docs for details: https://pnpm.io/workspaces."
```

Comment:
Same comment regarding trailing period and overall consistency of error formatting for these